### ISSUE TITLE
⌨ Screen readers don't announce content changes for filter instructor-led training page

### ISSUE THREAD

**Issue by NienkeBr** [GCTC-NTGC/gc-digital-talent#12551]
## ⌨ Accessibility Issue

Screen readers can't announce content changes for the applied filter on the instructor-led training page.

## 🦋 Expected Behaviour

Screen readers should announce the content changes after a filter has been applied.

## 🕵️ Details

https://talent.canada.ca/en/it-training-fund/instructor-led-training 

## 🙋‍♀️ Proposed Solution

> [!CAUTION]
> It seems our announcer is announce function has a bug so we are avoiding this for now until it is fixed in https://github.com/GCTC-NTGC/gc-digital-talent/issues/13558  

Refer to the notifications page for an example of how to do this.

- The filter option changes to: Regular text saying: "Filter by:" and then three filter options "All", "English", "French"
     - Filter by: All English French
     - Filtrer par: Tout Anglais Français
- Screen readers announce:
     - All: "All instructor-led training opportunities are shown."
     - Tout: "Toutes les possibilités de formation dirigées par un instructeur ou une instructrice sont affichées."
     - English: "Only English instructor-led training opportunities are shown."
     - Anglais: "Seules les possibilités de formation dirigées par un instructeur ou une instructrice en anglais sont affichées."
     - French: "Only French instructor-led training opportunities are shown."
     - Français: "Seules les possibilités de formation dirigées par un instructeur ou une instructrice en français sont affichées."

## ✅ Acceptance Criteria

- [ ] Filter has been updated with copy preceding the filter options.
- [ ] Screen readers announce selected filter.
- [ ] URL updates once filter is applied (instead of using useState)

**Comment by mnigh** [comment]
Possibly related: https://github.com/GCTC-NTGC/gc-digital-talent/issues/12454.
